projects/ion/src/lib/button/button.component.ts

```diff
@@ -70,5 +70,8 @@ export class IonButtonComponent implements OnInit {
 
   ngOnInit(): void {
     this.iconSize = ButtonIconSizeOptions[this.size];
+    if (this.disabled) {
```

Comment:
This works fine if the button already starts with disabled = true. But if disabled is passed as a variable time, and changes overtime, this will not solve the problem.

I made this example using your branch, where the switch is changing the variable disabled on the button (you can see the code here: https://angular-8-app-example-25v9aa.stackblitz.io):

[Screencast from 06-04-2023 15:17:26.webm](https://user-images.githubusercontent.com/57760325/230462518-309d04e7-b7dc-4fde-8b76-8aad2f9514fa.webm)


---
You'll probably need to do a solution using ngOnChanges, listening to the changes on disabled variable and then setting loading to false. Feel free to contact me for more explanation 😄 .

Reply:
I sent a email to you @mizek1